src/angular/partials/search-results.html

```diff
@@ -1,52 +1,39 @@
-<ul>
-  <li ng-repeat="result in results">
-    <span>{{result}}</span>
-  </li>
-</ul>
-
 <!-- view when clicked once a category/region is click on -->
 
 <main role="main">
-  <div class="col-sm-7">
-    map goes here
-  </div>
-  <div class ="col-xs-12 col-sm-5"> 
+  <div class ="col-xs-12 col-sm-12"> 
     <button class="btn icon-large icon-angle-left"> back</button>
     <h4>Search Results</h4>
     <ul class="list-group">
 
-      <a href="#" class="list-group-item">
-        <div class="row media-body">
-          <h2 class="media-heading">Name on the list</h2>
-          <img class="media-object pull-right" src="...">
-        </div>
-        <div class="row">
-          <div class="col-xs-5">
-            <h3>Hours</h3>
-          <p> Lorem Ipsum Dolor</p></div>
-          <div class="col-xs-7">
-            <h3>Activity Details</h3>
-          <p> Lorem Ipsum Dolor</p></div>
-          <div></div>
-        </div>
-      </a>
+      <div ng-repeat="result in results">
+        <a href="#" class="list-group-item">
+          
+          <!-- location name section -->
+          <div class="row media-body">
+            <h2 class="media-heading">{{result.properties.locationName}}</h2>
+            <img class="media-object pull-right" src="...">
+          </div>
+
+          <div class="row">
 
-      <a href="#" class="list-group-item">
-        <div class="row media-body">
-          <h2 class="media-heading">Name on the list</h2>
-          <img class="media-object pull-right" src="...">
-        </div>
-        <div class="row">
-          <div class="col-xs-5">
-            <h3>Hours</h3>
-          <p> Lorem Ipsum Dolor</p></div>
-          <div class="col-xs-7">
-            <h3>Activity Details</h3>
-          <p> Lorem Ipsum Dolor</p></div>
-          <div></div>
-        </div>
-      </a>
+            <!-- hours section -->
+            <div class="col-xs-5">
+              <h3>Hours</h3>
+              <p> {{ getOpeningTime(result) || 'N/A'}} - {{ getClosingTime(result) || 'N/A' }} </p>
```

Comment:
It really is just preference. I personally think it looks cleaner than a bunch of chained method which may confuse other people on the team at first glance.